jupyterlab.nix

```diff
@@ -53,9 +53,9 @@
       then
         echo jupyterWith needs to build JupyterLab.
         # we need to build the jupyter lab environment before it can be used
-        ${jupyterlabEnvBase}/bin/jupyter lab build
+        >&2 ${jupyterlabEnvBase}/bin/jupyter lab build
```

Comment:
This line shouldn't have `>&2`. It's a command for building the jupyterlab environment.

Reply:
thanks